src/politech_backend/settings/base.py

```diff
@@ -166,40 +166,24 @@
             'class': 'logging.StreamHandler',
             'formatter': 'verbose'
         },
-        'cronJobHandler': {
-            'level': 'DEBUG',
-            'class': 'logging.FileHandler',
-            'filename': '/code/src/logs/cronJobs.log',
-            'formatter': 'verbose'
-        },
     },
     'loggers': {
-        'cronJobs': {
-            'handlers': ['cronJobHandler'],
-            'propagate': False,
-            'level': 'DEBUG',
-        },
-        'cronJobServices': {
-            'handlers': ['cronJobHandler'],
-            'propagate': False,
-            'level': 'DEBUG'
-        },
-        'promise': {
+        'console': {
             'handlers': ['console'],
             'propagate': False,
-            'level': 'DEBUG',
+            'level': 'DEBUG'
         },
         'django.db.backends': {
-            'level': 'ERROR',
+            'level': 'DEBUG',
             'handlers': ['console'],
             'propagate': False,
         },
     }
 }
 
 CRON_CLASSES = [
-    "promises.cron_jobs.ConnectBillsAndPromises",
     "case_gather.cron_jobs.GatherCases",
+    "promises.cron_jobs.ConnectBillsAndPromises",
```

Comment:
Important change is important 🙈

Reply:
It runs in that over so best to have GatherCases first :)